test_scripts/API/CancelInteraction/005_SendCancelInteraction_Other_HMIReceivesNoRequest.lua

```diff
@@ -0,0 +1,80 @@
+---------------------------------------------------------------------------------------------------
+--  Precondition: 
+--  1) app1 is registered on SDL
+--
+--  Steps:
+--  1) app1 sends a sends an Alert RPC
+--  2) app1 sends a CancelInteraction Request with an INVALID functionID
+--
+--  Expected:
+--  1) the HMI does not receive the CancelInteraction request
+--  2) app1 receives INVALID_ID CancelInteraction response
+--  2) app1 receives SUCCESS from the Alert
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local common = require('user_modules/sequences/actions')
+
+--[[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+--[[ Local variables ]]
+local rpcInteraction = {
+  name = "Alert",
```

Comment:
Add `cancelID`

Reply:
added in https://github.com/smartdevicelink/sdl_atf_test_scripts/pull/2206/commits/471e8b5b7364595f4a5d0d0cb371f70f2d31a730